rdrf/rdrf/templatetags/password_reset_timeout.py

```diff
@@ -0,0 +1,45 @@
+from dateutil.relativedelta import relativedelta
+from django import template
+from django.conf import settings
+from django.utils.translation import gettext as _
+
+register = template.Library()
+
+
+def _pluralize(count: int, singular: str, plural: str):
+    return f"{count} {plural if count != 1 else singular}"
+
+
+def _readable_timedelta(delta):
```

Comment:
I don't think this will work for languages that have more complicated pluralisation like Arabic & Russian.

It seems there is a `naturaltime` template tag that we can use: https://docs.djangoproject.com/en/4.2/ref/contrib/humanize/#naturaltime

As well as the built-in template tag `timeuntil`

I've tried them out in this patch: [patch.txt](https://github.com/user-attachments/files/18484652/patch.txt)

It doesn't work for timedeltas, only future datetimes. There's also a 3rd-party humanization library we could use if this doesn't work properly.